build.sbt

```diff
@@ -31,7 +31,7 @@ val diffsonVersion = "4.6.0"
 val literallyVersion = "1.1.0"
 val weaverVersion = "0.8.4"
 
-ThisBuild / tlBaseVersion := "1.11"
+ThisBuild / tlBaseVersion := "1.12"
```

Comment:
This should not be changed. This is only changed after a release. The base version is the latest published one on which your changes are based. And this still is the 1.11.x series.

Reply:
My bad. Thanks for the catch :slightly_smiling_face:

Reply:
No worries, the build is a bit arcane at times.